docs/basic_syntax/functions.md

```diff
@@ -52,6 +62,22 @@ fun failing(name) {
 
 Notice that using `?` operator is automatically failing with the `status` code of the failing operation.
 
+You can mark a whole function as failable using `?` in the function declaration, an example:
+
+```ab
+fun failable(): Num? {
+    if 0 > 5 {
+        fail 1
+    }
+
+    return 1
+}
+
+let a = failable() failed: echo "Failed"
+
+if a is Num: echo "Succeded"
```

Comment:
```suggestion
```

Reply:
why do you want to remove those lines?

Reply:
They are unnecessary to the explanation and diverge attention.

Reply:
In my opinion is a way to show how to use the function also if failable

Reply:
That's totally unrelated to the discussed topic, though. The way failable functions are used is already demonstrated separately.